packages/flutter_tools/test/general.shard/base/logger_test.dart

```diff
@@ -73,6 +74,112 @@ void main() {
     ), isA<AppRunLogger>());
   });
 
+  testWithoutContext('DelegatingLogger delegates', () {
```

Comment:
For testing, can you make this use a Fake instance or perhaps use a BufferLogger instance instead?

We're still deciding what exactly we're going to do with mockito wrt to null-safety, the code generation changes are not really palatable so avoiding adding more tests which heavily depend on it will help us out a ton

Reply:
Ugh.  Using ` Mock` here was useful since it can verify the arguments (and it caught an argument that wasn't forwarded by the code I move from `DelegateLogger`).  I'll see if I can come up with something else.